R/assertions.R

```diff
@@ -783,46 +784,66 @@ assert_s3_class <- function(arg, cls,
 #'   assert_list_of(list, "numeric", named = TRUE)
 #' }
 #' try(example_fun2(list(1, 2, 3, d = 4)))
-assert_list_of <- function(arg, class, named = FALSE, optional = TRUE) {
-  assert_character_scalar(class)
+assert_list_of <- function(arg, cls,
```

Comment:
I think we should add an item to the "Breaking Changes" section of the Changelog regarding the renaming of the `class` argument.

Reply:
that's a good point. in this issue (https://github.com/pharmaverse/admiraldev/issues/367) we have a note to update the NEWS when we're all done, and i've added that we need to include this argument renaming in that entry as well.